src/components/dictionaryConcepts/components/CreateMapping.jsx

```diff
@@ -60,7 +71,10 @@ class CreateMapping extends Component {
 
   doSearch = async (inputValue, url) => {
     const options = await fetchSourceConcepts(INTERNAL_MAPPING_DEFAULT_SOURCE, inputValue, url);
-    this.setState({ options, isVisible: true });
+    // eslint-disable-next-line no-underscore-dangle
+    if (this._isMounted) {
+      this.setState({ options, isVisible: true });
```

Comment:
isVisible should have an inicial value in the state definition of the object.

Reply:
It already has an initial value in the state definition.